src/messages/MMDSBeacon.h

```diff
@@ -40,6 +40,7 @@ enum mds_metric_t {
   MDS_HEALTH_CLIENT_LATE_RELEASE_MANY,
   MDS_HEALTH_CLIENT_OLDEST_TID,
   MDS_HEALTH_CLIENT_OLDEST_TID_MANY,
+  MDS_HEALTH_CLIENT_LAGGY,
```

Comment:
```suggestion
  MDS_HEALTH_OSD_LAGGY,
```

Reply:
The client is perhaps responsive to the MDS but it just can't complete its work with the OSDs. Saying the clients are laggy is not accurate for why the MDS is not evicting the clients.

Reply:
makes sense, will change this

Reply:
done